challenges/classes.js

```diff
@@ -1,4 +1,56 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker {
+    constructor(lengthWidthHeight) {
+    this.length = lengthWidthHeight.length;
+    this.width = lengthWidthHeight.width;
+    this.height = lengthWidthHeight.height;
+  }
+  volume() {
+    return `Volume ${this.length * this.width * this.height}`;
+  };
+  surfaceArea () {
+    return `Surface Area: ${2 * (this.length * this.width + this.length * this.height + this.width * this.height)}`;
+  };
+}
+  
+  
+  const myCube = new CuboidMaker({
+    length: 10, 
+    width: 10, 
+    height:10
+  });
+  
+  const myCube2 = new CuboidMaker({
+    length: 100, 
+    width: 150, 
+    height:110
+  });
+
+  const myCube3 = new CuboidMaker({
+    length: 4, 
+    width: 5, 
+    height:5
+  });
+
+  console.log(myCube.volume()); // 100
+console.log(myCube.surfaceArea()); // 130
+console.log(myCube2.volume()); // 100
+console.log(myCube2.surfaceArea());
+console.log(myCube3.volume()); // 100
+console.log(myCube3.surfaceArea());
+
+
+
+
+
+
+
+
+
+
+
+
+
 
```

Comment:
make sure you read these because when I check your code this is the first thing I check to see if it works. so uncomment these and make sure they work